### PR TITLE
RL refactor

### PR DESCRIPTION
**Pull request by alibillalhammoud**

### CODE REVIEW

File: openfasoc/MLoptimization/test1.py
Comment (chetanyagoyal) [inline openfasoc/MLoptimization/test1.py:1]: Remove this file?

**Review by chetanyagoyal — COMMENTED** [review]
Remove the test1.py. 
Otherwise LGTM!

**Comment by msaligane** [comment]
Is this ready for review?

**Comment by msaligane** [comment]
@alibillalhammoud

**Comment by alibillalhammoud** [comment]
@msaligane not yet. I am currently rerunning on a fresh machine to verify all package versions I have listed in the requirements file are correct. Should be ready by Monday.

**Comment by alibillalhammoud** [comment]
Ok all good now. This PR is ready to merge

File: openfasoc/MLoptimization/test1.py
Comment (chetanyagoyal) [inline openfasoc/MLoptimization/test1.py:1]: Can you remove or refactor this file formally?

**Review by chetanyagoyal — CHANGES_REQUESTED** [review]

**Comment by chetanyagoyal** [comment]
Otherwise lgtm!

**Comment by chetanyagoyal** [comment]
@msaligane this can be merged.

**Review by chetanyagoyal — APPROVED** [review]

**Pull request closed without merging**